qemu/tests/live_snapshot_basic.py

```diff
@@ -51,7 +51,8 @@ def get_snapshot_file(self):
         """
         Get path of snapshot file.
         """
-        snapshot_file = "images/%s" % self.snapshot_file
+        image_format = self.params["image_format"]
+        snapshot_file = "images/%s.%s" % (self.snapshot_file, image_format)
```

Comment:
@yanan-fu It is not bug, it can operate correctly for previous test cases, but not fit for this one, it will cause error when run the test cases included in this patch, so must correct it, and current method is more expensible as well.